CLI tools/CurseForge to Packwiz-Modrinth.py

```diff
@@ -93,30 +94,43 @@ def main():
             toml.dump(pack_toml, f)
 
         # Packwiz import
-        os.system(packwiz_exe_path + ' curseforge import "' + cf_zip_path + '"')
+        os.system(f'{packwiz_exe_path} curseforge import "{cf_zip_path}"')
         if hydrogen:
-            os.system(packwiz_exe_path + " remove hydrogen")
-            os.system(packwiz_exe_path + " mr install hydrogen")
+            os.system(f"{packwiz_exe_path} remove hydrogen")
+            os.system(f"{packwiz_exe_path} mr install hydrogen")
         if modrinth_overrides:
-            os.system(packwiz_exe_path + " remove entityculling")
-            os.system(packwiz_exe_path + " mr install entityculling")
+            os.system(f"{packwiz_exe_path} remove entityculling")
+            os.system(f"{packwiz_exe_path} mr install entityculling")
 
     elif refresh_only:
-        os.system(packwiz_exe_path + " refresh")
+        os.system(f"{packwiz_exe_path} refresh")
 
     # Copy fresh manifest/modlist to git
     if not is_legacy and not refresh_only:
         extract_file(cf_zip_path, "manifest.json", git_path + "CurseForge", "CurseForge manifest.json", "Git")
         extract_file(cf_zip_path, "modlist.html", git_path + "CurseForge", "CurseForge modlist.html", "Git")
 
-    # Export packwiz pack via mmc-export method
+    # Export Packwiz pack via mmc-export method
     if mmc_export_packwiz_export and not refresh_only:
         mmc_zip_root = str(Path(cf_zip_path).parents[0])
         mmc_zip_path = mmc_zip_root + "\\Fabulously Optimized " + pack_version + ".zip"
         packwiz_config = git_path + "Packwiz\\mmc-export.toml"
 
-        cmd = f'mmc-export -i "{mmc_zip_path}" -f packwiz --modrinth-search loose -o "{mmc_zip_root}" -c "{packwiz_config}" -v {pack_version} --provider-priority Modrinth CurseForge Other --scheme mmc_export_packwiz_output'
-        os.system(cmd)
+        os.system(
+            " ".join(
+                (
+                    "mmc-export",
+                    f'-i "{mmc_zip_path}"',
+                    "-f packwiz",
+                    "--modrinth-search loose",
+                    f'-o "{mmc_zip_root}"',
+                    f'-c "{packwiz_config}"',
+                    f"-v {pack_version}",
+                    "--provider-priority Modrinth CurseForge Other",
+                    "--scheme mmc_export_packwiz_output",
+                )
+            )

```

Comment:
Did you know about [shlex.join](https://docs.python.org/3/library/shlex.html#shlex.join)?

Reply:
Now i know. Thanks for the tip :slightly_smiling_face: